docs/quickstarts/README.md

```diff
@@ -40,21 +40,21 @@ You can use this simple [preview tool](https://quickstarts-content-preview.surge
 
 
 ## Summary of steps
-This is an overview of the steps you will need to complete to publish an interactive quick start or Learning resource tile in the Hybrid Cloud Console.
+This is an overview of the steps you will need to complete to publish an interactive quick start or Learning resource card in the Hybrid Cloud Console.
 
 1. Write your draft quick start content, and get review and approval from product stakeholders.
 
 2. When the content is ready, create your YAML quick start files in a new directory in the `docs` directory of [**Red Hat Insights quickstarts**](https://github.com/RedHatInsights/quickstarts/tree/main/docs/quickstarts), following the detailed instructions in this `README.md` file. 
 
-3. Get your quick start or Learning resource tile added to the [Hybrid Cloud Console](https://console.redhat.com):
+3. Get your quick start or Learning resource card added to the [Hybrid Cloud Console](https://console.redhat.com):
  
     a. Create a request in [the RHCLOUD Jira project](https://issues.redhat.com/projects/RHCLOUD/issues/) for engineering to merge the content. Add the `platform-experience-services` label to your Jira.
```

Comment:
add the colleagues that need to be @ on this work.

Reply:
After discussing with the PMs, that list of contacts will remain in the Gdoc/Confluence page as this is a public repo https://docs.google.com/document/d/1KZfvvxtUzswojI5XfCgoqfEJnnRaLyl5xGlev6Bm-Rs/edit?tab=t.0#heading=h.jjeosz2sc2y7